bin/run.sh

```diff
@@ -39,7 +39,7 @@ if jq --exit-status '.custom?."test-in-release-mode"?' "$solution_path"/.meta/co
     release="--release"
 fi
 
-cargo +nightly test \
+timeout -v 5s cargo +nightly test \
```

Comment:
Our internal time out for the test runner is 20 seconds. I think we'll probably want to be somewhat close-ish to that timeout limit. Maybe crank this up to 15 seconds? If there are no objections, I'm happy to merge this after that change.

Reply:
:heavy_check_mark: Done